platform/darwin/src/MGLStyle.h

```diff
@@ -610,6 +610,16 @@ MGL_EXPORT
  */
 @property (nonatomic, strong) MGLLight *light;
 
+#pragma mark Managing Style's localization
+
+/**
+ Set to true to determine the localization according to the languages
+ supported by <a href="https://www.mapbox.com/vector-tiles/mapbox-streets-v7/#overview">Mapbox Streets</a>, 
+ it will look for the Preferred Language Order setting in the Language & Region section of Settings. 
+ If it doesn't fing a match it will fall back to english localization.
```

Comment:
This is a good start. A few notes:

- Wrap at 80 characters, if possible.
- I’m not sure if we follow this everywhere, but to avoid the Swift/ObjC `true`/`YES` issue, we can choose different words, like “enable”.
- I would make it more clear that we’re looking at the platform’s language setting — it might be good to toss in a “system’s” or “platform” qualifier somewhere. This is a darwin header, so it may be trickier.
- Make it clear that the Mapbox Streets in this case is the vector tile source, not the style of the same name.
- What happens if the style does not have the Mapbox Streets source?